public/wp-content/themes/glific/src/scss/header/style.scss

```
@@ -66,3 +66,74 @@
 		}
 	}
 }
+
+.desktop-menu {
+	li{
+		@extend .position-relative;
+	}
+	li a {
+		@extend .font-heebo-regular, .fz-md-16, .leading-md-20, .text-theme-dark-green, .text-decoration-none, .w-full;
+	}
+	.sub-menu {
+		@extend .bg-white, .position-absolute, .px-md-3, .mt-28, .w-220, .py-3, .left-n12\.5, .rounded-11;
+		display: none;
+		li {
+			@extend .border-bottom, .border-theme-pewter;
+			a{
+				@extend .w-full, .py-4, .pl-11, .d-block, .fz-14;
+				:hover {
+					@extend .text-theme-dark-green, .bg-theme-pewter;
+				}
+			}
+			a:hover{
+				@extend .text-theme-dark-green, .bg-theme-gin;
+			}
+		}
+		box-shadow: 6px 6px 9px -2px rgba(0,0,0,0.10);
+	}
+	.menu-item-has-children {
+		> a::after {
+			content: url('./dist/images/icon-dropdown.svg');
+			margin-left: 5px;
+		}
+		> .open::after {
+			content: url('./dist/images/icon-arrow-up.svg');
+			margin-left: 5px;
+		}
+	}
+}
+.sub-menu {
+	@extend .list-unstyled, .pt-md-2;
+}
+
+.mobile-primary-menu {
+	li{
+		@extend .position-relative;
+	}
+	.sub-menu {
+		@extend .bg-theme-secondary, .position-absolute, .px-md-3, .w-220, .py-3, .rounded-11, .right-0;
+		display: none;
+		li {
+			@extend .border-bottom, .border-theme-pewter, .mb-0, .px-2;
+			a{
```

@mohitgusain Can you please use any SCSS formatted? There are lots of formatted available in the market place.

```suggestion
			a {
```